.gitlab/ci/.gitlab-ci.global.yml

```
@@ -210,6 +210,31 @@
   - set | grep -E "^CI_.*=|^GITLAB.*=" | sort
   - section_end "Gitlab CI Build Parameters"
 
+.checkout-upload-commit-content-nightly: &checkout-upload-commit-content-nightly
+  - section_start "Checkout upload commit content nightly"
+  - | 
+    if [[ -n "${NIGHTLY}" && "${CI_COMMIT_BRANCH}" == "install_packs_nightly_bugs" ]]; then
```

```suggestion
    if [[ -n "${NIGHTLY}" && "${CI_COMMIT_BRANCH}" == "master" ]]; then
```

after finishing up the testing it should be `master`